nl/guides/teachers/grading/README.md

```diff
@@ -0,0 +1,117 @@
+
+Elk van de icoontjes is aanklikbaar en brengt je naar de corresponderende oplossing van die student voor die oefening.
+
+![evaluatie overzicht](./evaluatie-overzicht.png)
+
+### Puntenoverzicht
+
+Als je er voor gekozen hebt om punten te geven, dan is het puntenoverzicht ook zichtbaar. Analoog aan het detailoverzicht is in deze tabel ook een vakje per student per oefening voorzien. In elke vakje wordt de score weergegeven die je aan de corresponderende oplossing gaf. Indien nog geen punten voor een oplossing werden ingegeven, dan krijg je een streepje te zien. Indien nog niet voor alle scoreonderdelen een punt werd ingegeven, dan wordt de score in het lichtgrijs afgebeeld. Ook hier is elke score aanklikbaar en kom je zo bij de corresponderende oplossing terecht.
+
+De totaalscore per student en het gemiddelde voor elke oefening worden automatisch berekend.
+
+In de rechterbovenhoek van het puntenoverzicht is een downloadknop zichtbaar. Als je hier op klikt, dan wordt een csv-bestand met de punten gedownload. Dit bestand kan je eenvoudig open in bijvoorbeeld Excel.
+
+![evaluatie punten](./evaluatie-punten.png)
+
+## Een oplossing evalueren
+
+Op de evaluatiepagina krijg je steeds de laatst ingediende oplossing voor de deadline van een student voor een bepaalde oefening te zien.
+
+Aan de linkerkant zie je de status van de testen en de ingediende code. Op de code kan feedback worden toegevoegd aan een bepaalde regel door op het regelnummer te klikken. Voor algemene feedback klik je op "Globale opmerking toevoegen".
+
+**Merk op**: de feedback die je hier geeft is niet automatisch zichtbaar voor de studenten. Feedback binnen een evaluatie moet je steeds vrijgeven via de "Feedback vrijgeven" knop op het evaluatieoverzicht.
+
+Aan de rechter kant van de pagina vind je de navigatie en score interface. Van boven naar onder zie je volgende elementen:
+* **Volgende**: via deze knop ga je snel naar de volgende student voor dezelfde oefening.
+* **Punten geven**: Hier zie je alle score-items voor deze oefening. Je kan een score geven voor elk van de onderdelen. Je punten worden automatische bijgehouden als je ze intikt en ook de totaalscore is zo steeds up to date. Als alle deelscores zijn ingegeven, dan verschijnt een vinkje onder **Totaal**: Via het duimpje omhoog en omlaag kan je snel een 0 of de maximumscore op een onderdeel invullen.
+* **Voortgang voor deze oefening**: Dit is dezelfde voorgangsindicator als op de overzichtpagina. Het balkje geeft weer hoe ver je staat in het verbeterproces voor deze oefening.
+
+* **Status van de student**: Hier zie je de status van de ingediende oplossingen voor alle oefeningen binnen deze reeks. Deze icoontjes zijn klikbaar en brengen je snel naar de corresponderende evaluatiepagina.
+
+::: tip Gebruik de demo modus voor anoniem verbeteren
```

Comment:
(this should be "demomodus", but since we also call it "demo modus" on Dodona, we should keep it like this for now)